storage.go

```diff
@@ -312,11 +312,17 @@ func (s *Storage) nextPartObjectPageByPrefix(ctx context.Context, page *ObjectPa
 func (s *Storage) nextPartPage(ctx context.Context, page *PartPage) error {
 	input := page.Status.(*partPageStatus)
 
-	output, err := s.bucket.ListUploadedParts(oss.InitiateMultipartUploadResult{
+	imur := oss.InitiateMultipartUploadResult{
 		Bucket:   s.bucket.BucketName,
 		Key:      input.key,
 		UploadID: input.uploadId,
-	})
+	}
+
+	options := make([]oss.Option, 0)
```

Comment:
We can set the cap of slice here to avoid more memory allocate if content is decided, e.g. `make([]oss.Option, 0, 2)`.